### DISCUSSION TITLE
Missing events week of Sept. 16 and later

### DISCUSSION THREAD

**Discussion by myelhsa** [derekantrican/GAS-ICS-Sync#421] [Q&A] [answered]
The sync script is functioning properly - no errors in the log - but I updated the script yesterday just to be sure. Still appears to be functioning with no errors. My Outlook calendar will not sync the week of Sept. 16, 2024 and beyond. I added a bunch of appointments to the calendar yesterday for earlier in the year, and they all synced up until that date. I can't pinpoint the issue. Let me know what screenshots or script would be helpful to share so I can troubleshoot. Thanks.

**Comment by jonas0b1011001** [comment, marked as answer]
Hi, check out #125. It's an outlook limitation.

**Reply by myelhsa** [reply]
Got it. Bummer, but good to know. Still super grateful for the script - saves me from having to duplicate calendars. Thanks! So events will continue to sync as the 6-month window progresses, is that correct? It's not like I need to add appointments as "new" for the sync to be triggered. As long as events are there, whether they're existing or new, they will sync as they enter the 6-month window. Do I have all that straight?

**Reply by jonas0b1011001** [reply]
100% correct
